Neos.Flow/Documentation/TheDefinitiveGuide/PartII/Configuration.rst

```diff
@@ -181,16 +183,20 @@ Environment Variables
 
 Some specific flow behaviour can also be configured with a couple of environment variables.
 
-============================	==================================================
-Variable						Description
-============================	==================================================
-FLOW_ROOTPATH					Can be used to override the path to the Flow root.
-FLOW_CONTEXT					Use to set the flow context (see above).
-FLOW_PATH_TEMPORARY_BASE		Can be used to set a path for temporary data.
-FLOW_LOCKHOLDINGPAGE			Use to specify the html page shown when the site is locked.
-								This is relative to the Packages directory.
-FLOW_ONLY_COMPOSER_LOADER		Set to true (1) to only use composer autoloader.
-============================	==================================================
++---------------------------+------------------------------------------------------------+
+| Variable                  | Description                                                |
++===========================+============================================================+
+| FLOW_ROOTPATH             | Can be used to override the path to the Flow root          |
++---------------------------+------------------------------------------------------------+
+| FLOW_CONTEXT              | Use to set the flow context (see above)                    |
++---------------------------+------------------------------------------------------------+
+| FLOW_PATH_TEMPORARY_BASE  | Can be used to set a path for temporary data               |
++---------------------------+------------------------------------------------------------+
+| FLOW_LOCKHOLDINGPAGE      | Use to specify the html page shown when the site is locked |
+|                           | This is relative to the Packages directory                 |
++---------------------------+------------------------------------------------------------+
+| FLOW_ONLY_COMPOSER_LOADER | Set to true (1) to only use composer autoloader            |
```

Comment:
I think we wanted to remove this already with Flow 5 :D Maybe time to do so, or at least set this to default = true
/cc @kitsunet

Reply:
Good point.. I missed that deprecation because it didn't have the `@deprecated` annotation.
I'll merge this one as-is, but we should adjust it with #2288 if we agree to that